timesketch/frontend-ng/src/views/Home.vue

```diff
@@ -15,6 +15,45 @@ limitations under the License.
 -->
 <template>
   <div>
+    <!-- Top horizontal toolbar -->
+    <v-toolbar flat color="transparent">
+      <v-avatar class="mt-2 ml-n3">
+        <router-link to="/">
+          <v-img src="/dist/timesketch-color.png" max-height="25" max-width="25" contain></v-img>
+        </router-link>
+      </v-avatar>
+      <span style="font-size: 1.2em">timesketch</span>
+
+      <v-spacer></v-spacer>
+      <v-btn small depressed v-on:click="switchUI"> Use the old UI </v-btn>
+      <v-avatar color="grey lighten-1" size="25" class="ml-3">
+        <span class="white--text">{{ currentUser | capitalize }}</span>
```

Comment:
This will display the whole username and overflow the v-avatar circle. I would recommend to either use only initials or dynamically adjust the v-avatar field (e.g. as a pill).

Reply:
I agree - I plan on fixing this in an upcoming PR.